scandipwa-paypal/src/app/component/PayPal/PayPal.style.scss

```diff
@@ -10,8 +10,11 @@
  */
 
 .PayPal {
-    margin-top: 2rem;
     min-height: 5rem;
+    
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/manual/ScandiPWA/base-theme/pullRequest?prid=5162789)